eqcorrscan/core/match_filter.py

```diff
@@ -51,6 +52,31 @@
 from eqcorrscan.core.lag_calc import lag_calc
 
 
+def _spike_test(stream, percent=0.99, multiplier=1e6):
```

Comment:
Is this for avoiding [bottleneck: 164](https://github.com/kwgoodman/bottleneck/issues/164) ? It might be worth adding a bit in the doc string about why this function might be used.